src/core/mongodb.tf

```
@@ -52,8 +52,8 @@ module "cosmosdb_account_mongodb" {
   tags = var.tags
 }
 
-resource "azurerm_cosmosdb_mongo_database" "mongodb" {
-  name                = format("%s-cosmosdb-mongodb", local.project)
+resource "azurerm_cosmosdb_mongo_database" "selcProduct" {
```

```suggestion
resource "azurerm_cosmosdb_mongo_database" "selc_product" {
```
from terraform lint rules